src/main/scala/io/iohk/ethereum/domain/BlockHeader.scala

```diff
@@ -58,13 +60,19 @@ case class BlockHeader(
 
 object BlockHeader {
 
+  private val NumberOfFields = 16
+
   val emptyOmmerHash = ByteString(Hex.decode("1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"))
 
   def getEncodedWithoutNonce(blockHeader: BlockHeader): Array[Byte] = {
     val rlpEncoded = blockHeader.toRLPEncodable match {
-      case rlpList: RLPList =>
+      case rlpList: RLPList if rlpList.items.length == (NumberOfFields - 1) =>
```

Comment:
i prefer something like: `if (isPreviousToEcip1089(rlpList) =>` WDYT?

Reply:
I found a way I preferred most, to not require the NumberOfFields value, I'll soon push it